web-previewer/index.html

```diff
@@ -6,6 +6,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
     <title>Developer DAO: Avatar Preview</title>
```

Comment:
```suggestion
```

We need to remove this old `<title>` tag! 😄

Reply:
ha! good call, sorry I missed that one. 👍